src/accel/cuda/GPUSolver.h

```diff
@@ -14,22 +14,33 @@
 #include "../../Solver.h"
 #endif
 
-#define PySys_WriteStdout printf
-
-#include <thrust/reduce.h>
 #include <thrust/device_vector.h>
+#include <thrust/copy.h>
+#include <thrust/fill.h>
+#include <thrust/reduce.h>
+#include <thrust/replace.h>
+#include <thrust/functional.h>
+#include <thrust/iterator/constant_iterator.h>
 #include <sm_20_atomic_functions.h>
 #include "clone.h"
 #include "GPUExpEvaluator.h"
 
+#define PySys_WriteStdout printf
```

Comment:
Can you move this line back to before the include statements. Moving this line to after the include statements produced the error included below when compiling with CUDA 7. It compiled just fine with CUDA 6 though. When I moved this line back to before the include statements, it compiled just fine with CUDA 6 and 7.

/usr/local/cuda/include/thrust/system/cuda/detail/bulk/detail/terminate.hpp(45): error: namespace "std" has no member "PySys_WriteStdout"

1 error detected in the compilation of "/tmp/tmpxft_00001715_00000000-10_GPUSolver.compute_30.cpp1.ii".
error: command 'ccache' failed with exit status 2